scripts/jest/config.build-devtools.js

```diff
@@ -49,7 +50,10 @@ moduleNameMapper['^react-reconciler/([^/]+)$'] =
 
 module.exports = Object.assign({}, baseConfig, {
   // Redirect imports to the compiled bundles
-  moduleNameMapper,
+  moduleNameMapper: {
+    ...devtoolsRegressionConfig.moduleNameMapper,
```

Comment:
nit: maybe swap these two lines as `devtoolsRegressionConfig` is a special case?